### DISCUSSION TITLE
Chakra + Stitches version

### DISCUSSION THREAD

**Discussion by nelsieborja** [chakra-ui/chakra-ui#4422] [Ideas]
I noticed most of the main features (eg: variants, tokens, styled-system, CSS variables) of Chakra are built-in features of [Stitches](https://stitches.dev/), it is as if these 2 libraries were made for each other like a perfect match. Plus Chakra can also benefit from Stitches' atomic CSS and low-runtime. That said, would be nice to have this version of the library.

**Comment by nelsieborja** [comment]
Any update on this? There is a new yet another React UI library called [NextUI](https://nextui.org/) which is based on Stitches. Just sharing coz it is worth checking out.

**Reply by paulbalogh** [reply]
Chakra UI's maturity and the low-runtime of Stitches would make a great combination. Next UI is still very much in its early days, not really usable for our projects. Unlike Chakra, which we love a lot. So +1 from me.

**Comment by joeyfigaro** [comment]
Some attention here would be great. We're currently using `@radix-ui/*` primitives to build out a UI kit and it's breaking critically. Took a look at chakra to see if it'd be a suitable replacement, but it's unclear how stitches and chakra might work out.

**Reply by punkpeye** [reply]
@joeyfigaro @nelsieborja Did you figure out if Chakra works with Stitches?

**Reply by Jarrodsz** [reply]
The theme files of chakra ui look very close to stitches
